instrumentation/opentelemetry/init.go

```diff
@@ -245,6 +248,19 @@ func Init(cfg *config.AgentConfig) func() {
 // and returns a shutdown function to flush data immediately on a termination signal.
 func InitWithSpanProcessorWrapper(cfg *config.AgentConfig, wrapper SpanProcessorWrapper,
 	versionInfoAttrs []attribute.KeyValue) func() {
+	logger, err := zap.NewProduction()
```

Comment:
Is there existing log config we can use to set some config in the zap logger we create?

Reply:
We dont have any existing log config in the hypertrace proto. The traceable goagent initializes logger with this config: https://github.com/Traceableai/goagent-src/blob/main/init.go#L29. Maybe we can move that logic here? Or can just use the new zap compatible api there?

Reply:
There's a techdebt ticket to move the hypertrace goagent login into traceable goagent and archive hypertrace goagent. So we should probably wait until we do that and in hypertrace we can go with some zap logger support. But it should be overridable by the traceable goagent one for now. I did not know that we already had a zap logger in traceable goagent.

Reply:
Oh okay, then I think we should be good. We can pass that zap logger here with the new api

Reply:
My ask here is that we should be able to use the logger created in the agent(i.e. mirroring-agent/ebpf) so that we can control where do the logs go(file/stdout/stderr), loglevels, rotation from one place.

Reply:
Yes. As of now ebpf calls this api: https://github.com/Traceableai/ebpf/blob/main/pkg/export/exporter.go#L41
This calls the InitWithSpanProcessorWrapper here https://github.com/Traceableai/goagent-src/blob/main/init.go#L37. Now we'll update the goagent-src to also expose an api for taking in a zap logger, then we can update the respective clients (ebpf/mirroring-agent/ext_cap)

Reply:
Yeah. If the same zap logger or zapcore object can be used throughout, that would be ideal.